src/course-unit/data/thunk.js

```diff
@@ -239,3 +246,37 @@ export function fetchCourseVerticalChildrenData(itemId) {
     }
   };
 }
+
+export function copyToClipboard(usageKey) {
+  const POLL_INTERVAL_MS = 1000; // Timeout duration for polling in milliseconds
```

Comment:
[question]: If we plan to use notification delay for smoothness and improve the user experience, should we put this constant in the appropriate constants file?

Reply:
[clarify]: After you added a mandatory delay for notifications to show, should I still be able to see the notifications now?

https://github.com/raccoongang/frontend-app-course-authoring/assets/93188219/3ef3a6db-37aa-479b-b781-b331f4e76eeb